src/FormattedText.tsx

```diff
@@ -0,0 +1,246 @@
+import * as React from 'react'
+import * as Marked from 'marked'
+import * as He from 'he'
+
+
+interface Props {
+    text: string,
+    format: string
+}
+
+interface State {
+}
+
+export class FormattedText extends React.Component<Props, State> {
+
+    shouldComponentUpdate(nextProps: Props, nextState: State, nextContext: any): boolean {
+        return this.props.text !== nextProps.text || this.props.format !== nextProps.format
+    }
+
+    render(): React.ReactElement<any> {
+        const format = this.props.format || "markdown"
+        if (format === "plain") {
+            return this.renderPlainText()
+        } else if (format === "xml") {
+            return this.renderXml()
+        } else {
+            return this.renderMarkdown()
+        }
+    }
+
+    renderPlainText(): React.ReactElement<any> {
+        // TODO @eanders-MS: This is placeholder until updated to DirectLine 3.0
```

Comment:
Why a placeholder? We can test this now with FeatureBot. If you don't know about FeatureBot, ask Dan.